assets/airports/kstl.json

```diff
@@ -0,0 +1,13310 @@
+        "30L": [["WOLTR", "A25+"], "ANNII", "NATCA"],
+        "30R": [["WOLTR", "A25+"], "ANNII", "NATCA"]
+      },
+      "exitPoints": {
+        "SPI": ["SPI"]
+      },
+      "draw": [
+      	["ANNII", "NATCA", "SPI*"],
+      	["DSAPP", "NATCA"],
+      	["BEART", "NATCA"]
+      ]
+    },
+    "ROTRY3": {
+      "icao": "ROTRY3",
+      "name": "Rotary Three",
+      "rwy": {
+      	"6":   [],
+      	"11":  [["FUNKE", "A25+"], "BEART", "GMSKI", "ROTRY"],
+      	"12L": [["FUNKE", "A25+"], "BEART", "GMSKI", "ROTRY"],
+        "12R": [["FUNKE", "A25+"], "BEART", "GMSKI", "ROTRY"],
+        "24":  [],
+        "29":  [["DSAPP", "A25+"], "ROTRY"],
+        "30L": [["WOLTR", "A25+"], "ROTRY"],
+        "30R": [["WOLTR", "A25+"], "ROTRY"]
+      },
+      "exitPoints": {
+        "BDF": ["SKUTR", "BDF"]
+      },
+      "draw": [
+      	["GMSKI", "ROTRY", "SKUTR", "BDF*"],
+      	["WOLTR", "ROTRY"],
+      	["DSAPP", "ROTRY"]
+      ]
+    },
+    "TEDDD3": {
+      "icao": "TEDDD3",
+      "name": "Ted Three",
+      "rwy": {
+      	"6":   [],
+      	"11":  [["FUNKE", "A25+"], "RAMKR", "TEDDD"],
+      	"12L": [["FUNKE", "A25+"], "RAMKR", "TEDDD"],
+        "12R": [["FUNKE", "A25+"], "RAMKR", "TEDDD"],
+        "24":  [],
+        "29":  [["DSAPP", "A25+"], "ANNII", "LENXX", "KAEYE", "TEDDD"],
+        "30L": [["WOLTR", "A25+"], "ANNII", "LENXX", "KAEYE", "TEDDD"],
+        "30R": [["WOLTR", "A25+"], "ANNII", "LENXX", "KAEYE", "TEDDD"]
+      },
+      "exitPoints": {
+        "HILTS": ["ENL", "ZOLLI", "HILTS"],
+        "IIU":   ["ENL", "ZOLLI", "HILTS", "IIU"]
+      },
+      "draw": [
+      	["KAEYE", "TEDDD", "ENL", "ZOLLI", "HILTS*"],
+      	["RAMKR", "TEDDD"],
+      	["HILTS", "IIU*"]
+      ]
+    },
+    "WHRLI4": {
+      "icao": "WHRLI4",
+      "name": "Whirly Four",
+      "rwy": {
+      	"6":   [],
+      	"11":  [["LUUBE", "A25+"], "DAFUS", "TRAVS", "SEKAY", "LIISA", "WHRLI"],
+      	"12L": [["HEFNR", "A25+"], "DAFUS", "TRAVS", "SEKAY", "LIISA", "WHRLI"],
+        "12R": [["HEFNR", "A25+"], "DAFUS", "TRAVS", "SEKAY", "LIISA", "WHRLI"],
+        "24":  [],
+        "29":  [["HAGEE", "A25+"], "WHRLI"],
+        "30L": [["PYUNG", "A25+"], "WHRLI"],
+        "30R": [["PYUNG", "A25+"], "WHRLI"]
+      },
+      "exitPoints": {
+        "MCM": ["MRCOS", "MCM"]
+      },
+      "draw": [
+      	["LIISA", "WHRLI", "MRCOS", "MCM*"],
+      	["PYUNG", "WHRLI"],
+      	["HAGEE", "WHRLI"]
+      ]
+    }
+  },
+  "stars": {
+    "AARCH1": {
+      "icao": "AARCH1",
+      "name": "Arch One",
+      "entryPoints": {
+        "AKBAR": [["AKBAR", "A340-|S350"], ["PRUET", "A320-"], ["UKAAY", "A220|S280"]],
+        "RBS":   ["RBS", "JARKE", ["BASCO", "A370-"], ["GORDO", "A350-"], ["WLDBL", "A300-"], ["HONKK", "A220|S280"]],
+        "VHP":   ["VHP", ["AKBAR", "A340-|S350"], ["PRUET", "A320-"], ["UKAAY", "A220|S280"]]
+      },
+      "body": [["AARCH", "A180|S280"], ["PETTI", "A120"]],
+      "rwy": {
+        "6":   [],
```

Comment:
They are correct for the SID's since they should fly direct to the body (which I somehow forgot to add), but they should be removed from the STAR's